spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -839,6 +838,28 @@ We may specify that all mock objects are alternatives:
 public @interface Mock {}
 ----
 
+[[priority_stereotype]]
+
+===== Declaring stereotype with `@Priority`
+
+A stereotype may declare a `@Priority` annotation which functions as a means of enabling and ordering affected alternatives, interceptors and decorators.
+
+If there are two different stereotypes declared by the bean that declare different priority values, then there is no default priority and the bean must explicitly declare a `@Priority` annotation. If it does not explicitly declare priority, the container automatically detects the problem and treats it as a definition error.
+
+If a bean explicitly declares priority, any priority values declared by stereotypes are ignored.
+
+Following sample shows a stereotype that can be used to mark bean as globally enabled alternative:
+
+[source, java]
+----
+@Alternative
+@Priority(Interceptor.Priority.APPLICATION + 5)
+@Stereotype
+@Target(TYPE)
+@Retention(RUNTIME)
+public @interface TestReplacementStereotype {}
```

Comment:
We could just call it `@Mock`, following on the example from previous section? Or `@TestDouble`, which is essentially the same thing (or a more general term covering mocks, stubs, spies, fakes and I don't know what).